app/service/rest.py

```diff
@@ -531,3 +539,32 @@ def get_yearly_monthly_usage(service_id):
         return json.dumps(json_results)
     except TypeError:
         return jsonify(result='error', message='No valid year provided'), 400
+
+
+@service_blueprint.route('/<uuid:service_id>/inbound-api', methods=['POST'])
+def create_service_inbound_api(service_id):
+    data = request.get_json()
+    validate(data, service_inbound_api)
+    data["service_id"] = service_id
```

Comment:
It's difficult to test this, if it does timeout, that will timeout the request for the admin app. 
Instead we could add a test button from the admin app.